src/components/ebay-skeleton/index.marko

```diff
@@ -0,0 +1,46 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var ignoredAttributes = [
+    "a11yText",
+    "class",
+    "size",
+    "type",
+    "multiline",
+    "count",
+    "columns",
+];
+static var validTypes = ["avatar", "button", "text", "form", "image", "tile"];
+static var validSizes = ["large", "small"];
+static var validTypeSizes = ["button", "text", "tile"];
+$ var size = validSizes.includes(input.size) ? input.size : "default";
+$ var type = validTypes.includes(input.type) ? input.type : "form";
+$ var typeClass = `skeleton-${type}`;
+$ var skeletonClass = (
+    validTypeSizes.includes(input.type) &&
+    (size !== "default"
+        ? `${typeClass}--${size}`
+        : type === "text"
+        ? `${typeClass}--small`
+        : "")
+);
+
+<div
+    ...processHtmlAttributes(input, ignoredAttributes)
+    class=[
+        typeClass,
+        skeletonClass,
+        input.multiline && type === "text" && "skeleton-text--multiline",
+        input.class,
+    ]
+    role="img"
+    aria-label=input.a11yText || "Loading"
+>
+    <if(type === "tile")>
```

Comment:
I don't know if I like having this as an `input` option, because composite components won't look the same for every use case. It would be nice as an example, but I don't think it should be baked into the component like this.

Reply:
No longer a component. Moved to composite example.